dot-studio/protobuf

```diff
@@ -26,6 +26,7 @@ document "compile_go_protobuf" <<DOC
   ------------------------------------------------------------------
   #!/bin/bash
   set -x
+  GOPATH=$(go env GOPATH)
```

Comment:
[nit] I don't think we need this since the scaffolding is doing it for us.

Reply:
Hmm this is just an example. I've added it because _this way_, it's more likely to work outside of the scaffolding, for development purposes. Do we agree it doesn't hurt? 😉